codes/classical/q-ary_digits/skew_cyclic.yml

```diff
@@ -0,0 +1,119 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+# Use UTF-8 unicode encoding
+# AMS-TeX commands are rendered inside \( ... \) using MathJaX.
+# Allowed external bibliographic references are
+#    \cite{arXiv:#.#} or \cite{arXiv:quant-ph/#},
+#    \cite{doi:#}, or, as a last resort
+#    \cite{manual:{(enter citation line incl. author and year here)}}
+# External websites such as code tables, coding theory packages, github pages linked as
+#    \url{https://example.com/example}
+#    \href{https://example.com/example}{link text}
+# Internal references to codes are 
+#    \hyperref[code:code_id]{link text}
+# Delete instructional comments when submitting
+
+# code id, physical, logical are all lower case
+code_id: skew_cyclic
+physical: bits
+logical: bits
+
+# Apostrophes are denoted by two apostrophe characters, i.e., ''
+# Code title is singular
+name: 'Skew-Cyclic code'
+
+# First reference with the code (optional).
+introduced: '\cite{arXiv:math/0604603}'
+
+# Anything applicable to a larger parent set of codes (see below) should go in
+# that entry instead of here.
+description: 'Ideals in the polynomial ring \(F_q[X,\theta]/(X^n-1)\) where \(\theta\) is an isomorphism of \(F_q\).'
```

Comment:
Could use a more extended description in terms of what is being encoded into what, and how the logical strings correspond to ideals.

Reply:
Just as with quantum_cyclic codes, I was too deep into the paper and forgot the simple description. I updated to use the simple definition.